omicron/cli/process.py

```diff
@@ -741,9 +747,10 @@ def main(args=None):
                              "4000 seconds")
                 start = end - 4000
         else:
-            logger.debug("Online segment record recovered")
+            logger.debug(f"Online segment record recovered: {last_run_segment[0]} - {last_run_segment[1]}")
```

Comment:
Can probably just use the internal repr of the segment object itself:

```suggestion
            logger.debug(f"Online segment record recovered: {last_run_segment}")
```

Reply:
OK